stockflux-watchlist/public/notification.css

```diff
@@ -0,0 +1,81 @@
+body {
+  background-color: rgb(40, 49, 61);
+  height: calc(100vh - 2px);
+  width: calc(100vw - 2px);
+  border: 1px solid #45d6bc;
+  padding: 0;
+  margin: 0;
+  border-radius: 7px;
+  overflow: hidden;
+  display: inline-flex;
+}
+
+.notification-icon {
+  margin: auto;
+  border-right: 1.5px solid #45d6bc;
+}
+
+span#message,
+span#symbol,
+p#title {
+  color: lightgray;
+  font-family: 'Roboto', sans-serif;
```

Comment:
Do we get Roboto without importing from googlefonts?

Reply:
Nope, this is what happens when I remove it -> 
![image](https://user-images.githubusercontent.com/32611900/58241606-38c28b80-7d45-11e9-8688-90cdb3163670.png)

Reply:
I misunderstood you initially, sorry, here's the fix and thanks -> f035838fc3d63a4f22f7bc2367bf2d96fe61d96c.